### DISCUSSION TITLE
Hello, I would like to ask how to solve api.vrchat.cloud:443

### DISCUSSION THREAD

**Discussion by NekoYamaShiro** [vrcx-team/VRCX#817] [Q&A]
![2f11488aa51a4c02bda666508696d6de](https://github.com/vrcx-team/VRCX/assets/49232332/d0ed102c-1140-47ed-9dbb-2b4126fcae67)
I woke up this morning and I couldn't log in, so I saw that the popup in the bottom left was this command

**Comment by Natsumi-sama** [comment]
This problem could be due to VPN or firewall configuration, test the VRChat website to make sure your internet is working correctly.

**Reply by NekoYamaShiro** [reply]
But I don't have any vpn on

**Reply by NekoYamaShiro** [reply]
Ok, I'll try to log on to the vrchat website first
